web/beacon-app/src/components/common/TagState/TagState.tsx

```diff
@@ -0,0 +1,27 @@
+import { Trans } from '@lingui/macro';
+
+import { Tag } from '@/components/ui/Tag';
+import { TOPIC_STATE } from '@/constants/rolesAndStatus';
+
+interface TopicStateTagProps {
+  status: string;
+  colorScheme?: string;
+}
+
+// TODO: list all possible values of status
```

Comment:
The topic states currently returned from BE can be found here: https://github.com/rotationalio/ensign/blob/630bc531d1fc1ad58edd8e8a93eb7c9a7b599610/pkg/tenant/db/topics.go#L20-L25